ansible/files/airflow/entrypoint.sh

```diff
@@ -28,6 +29,27 @@ sed -i "s/{DB_USER}/${DB_USER}/" $AIRFLOW_HOME/airflow.cfg
 sed -i "s/{DB_PASSWORD}/${DB_PASSWORD}/" $AIRFLOW_HOME/airflow.cfg
 sed -i "s/{AIRFLOW_ENABLE_AUTH}/${AIRFLOW_ENABLE_AUTH}/" $AIRFLOW_HOME/airflow.cfg
 
+# Fetch the SMTP credentials
```

Comment:
Could we fetch the credentials from the ansible script and pass them as environment variables?